### PR TITLE
fix(TDP-3549): Error when trying to sort on Actions columns

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3549

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) Other information**:
Depends on https://github.com/Talend/ui/pull/335

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/5/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/5/)

File: dataprep-webapp/src/app/components/widgets-containers/inventory-list/inventory-list-container.spec.js [hunk 1]
```diff
@@ -480,5 +480,15 @@ describe('Inventory list container', () => {
 			expect(lastCallArgs.id).toBe('dataset:sort');
 			expect(lastCallArgs.type).toBe('@@dataset/SORT');
 		}));
+
+		it('should have an invisible and screen reader compatible header', inject(() => {
+				// when
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/widgets-containers/inventory-list/inventory-list-container.spec.js]: Can you adust indent here

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/7/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/7/)

File: dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/views/provider/ListViews.java [hunk 2]
```diff
@@ -84,7 +84,7 @@
             .list(
                     ListDetailsSettings.builder()
                             .column("name", "Name")
-                            .column("statusActions", "", "actions")
+                            .column("statusActions", "Actions", "actions")
```
Comment (frassinier) [inline dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/views/provider/ListViews.java]: It could be nice to replace 3rd arg in favor of a boolean to deal with its hidden option?

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/23/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/23/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/25/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/25/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/26/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/26/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/27/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/27/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/28/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/28/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/29/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/29/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/30/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/30/)

File: dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/views/api/list/ListDetailsSettings.java [hunk 3]
```diff
@@ -103,14 +105,28 @@ public static Builder builder() {
         private ListTitleSettings titleProps;
 
         public Builder column(final String key, final String label) {
-           this.column(key, label, null);
+            this.column(key, label, null, null);
+            return this;
+        }
+
+        public Builder column(final String key, final String label, final Boolean hideHeader) {
```
Comment (frassinier) [inline dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/views/api/list/ListDetailsSettings.java]: Why did you need it?

File: dataprep-api/src/test/java/org/talend/dataprep/api/service/AppSettingsAPITest.java [hunk 4]
```diff
@@ -20,9 +20,11 @@
 import static org.talend.dataprep.api.service.settings.actions.api.ActionSettings.PAYLOAD_ARGS_KEY;
 import static org.talend.dataprep.api.service.settings.actions.api.ActionSettings.PAYLOAD_METHOD_KEY;
 
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.hamcrest.Matchers;
```
Comment (frassinier) [inline dataprep-api/src/test/java/org/talend/dataprep/api/service/AppSettingsAPITest.java]: This import is useful for?

File: dataprep-api/src/test/java/org/talend/dataprep/api/service/AppSettingsAPITest.java [hunk 5]
```diff
@@ -20,9 +20,11 @@
 import static org.talend.dataprep.api.service.settings.actions.api.ActionSettings.PAYLOAD_ARGS_KEY;
 import static org.talend.dataprep.api.service.settings.actions.api.ActionSettings.PAYLOAD_METHOD_KEY;
 
+import java.util.HashMap;
```
Comment (frassinier) [inline dataprep-api/src/test/java/org/talend/dataprep/api/service/AppSettingsAPITest.java]: Useless I guess?

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/49/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/49/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/103/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/103/)
